subjects/java/checkpoints/perfect-number/README.md

```diff
@@ -0,0 +1,50 @@
+## Perfect Number
+
+### Instructions
+
+Write a program to check if a given number is a perfect number. A perfect number is a positive integer that is equal to the sum of its proper positive divisors, excluding itself.
+
+### Expected Class
+
+```java
+public class PerfectNumber {
+    public boolean isPerfectNumber(int number) {
+        // Implementation to check if the given number is a perfect number
+    }
+}
+```
+
+### Usage
+
+Here is a possible `ExerciseRunner.java` to test your class:
+
+```java
+public class ExerciseRunner {
+    public static void main(String[] args) {
+        PerfectNumber perfectNumber = new PerfectNumber();
+
+        // Test case 1
```

Comment:
Not a big fan of those comments, but this is just an opinion, we don't see this in any of the other piscines.

Reply:
removed